src/main/scala/beam/utils/data/ctpp/JointDistribution.scala

```diff
@@ -88,9 +87,9 @@ class JointDistribution(
       new CPair[Map[String, String], java.lang.Double](row(value, sampleWithinRange), value(RETURN_COLUMN).toDouble)
     }.toVector
 
-    val values = pmf.map(_.getValue)
-    if (values.isEmpty || values.reduce(_ + _) == 0.0) {
-      return Map()
+    val values = pmf.map(_.getValue.doubleValue())
```

Comment:
Another slot where the express value call seems duplicative.

Reply:
@JustinPihony pls check explanation above